packages/entities/entities-plugins-metadata/src/index.ts

```diff
@@ -425,6 +425,14 @@ export const PLUGIN_GROUP_AND_SCOPE_MAP: Record<string, {
     group: PluginGroup.AI,
     scope: [PluginScope.GLOBAL, PluginScope.SERVICE, PluginScope.ROUTE, PluginScope.CONSUMER, PluginScope.CONSUMER_GROUP],
   },
+  'solace-upstream': {
```

Comment:
Should the plugin name be `solace-producer`: https://github.com/Kong/kong-ee/blob/4011a7009eca55aa7f33d22825ab20c8cc9459a2/kong/plugins/solace-producer/schema.lua#L15 ?